src/desktop/apps/artsy-v2/apps/collection/collectionMiddleware.ts

```diff
@@ -0,0 +1,23 @@
+import { NextFunction } from "express"
+import { userHasLabFeature } from "v2/Utils/user"
+import { collectionToArtistSeriesSlugMap } from "v2/Apps/Collect/Utils/collectionToArtistSeriesSlugMap"
+
+export const handleCollectionToArtistSeriesRedirect = async (
+  req,
+  res,
+  next: NextFunction
+) => {
+  const user = res.locals.sd.CURRENT_USER
+  const collectionSlug = req.params.collectionSlug
+
+  if (userHasLabFeature(user, "Artist Series")) {
+    let seriesSlug: string
+    if ((seriesSlug = collectionToArtistSeriesSlugMap[collectionSlug])) {
```

Comment:
typo?

Reply:
The single equals you mean? I'm assigning it and checking the value of the assignment in the same line. Just short-hand to save a line, I suppose 😅

Reply:
hah, oldskooool 🙂 might be a lil clearer if this kind of assignment was avoided tho.